docs/replication_setup.md

```diff
@@ -48,7 +48,7 @@ spec:
         spec:
           containers:
             - name: clickhouse-pod
-              image: clickhouse/clickhouse-server:22.3
+              image: clickhouse/clickhouse-server:23.8
```

Comment:
sync with name in line 47 and find usages this template and change template name reference too